package.json

```diff
@@ -31,6 +31,7 @@
   },
   "homepage": "https://github.com/ditrit/terrator-plugin#readme",
   "dependencies": {
+    "ajv": "^8.12.0",
```

Comment:
why package-lock.json updated is missing from this commit ?

Reply:
Because I update it in the next commit